### ISSUE TITLE
MvcReportViewer change export filename with default export button

### ISSUE THREAD

**Issue by cuongbk81** [ilich/MvcReportViewer#162]
I use mvcreportviewer for my solution, i config as:

@Html.MvcReportViewerFluent("App_Data/Report/TestReport.rdlc").ProcessingMode(ProcessingMode.Local).ReportParameters(new { ReportName = Model.ReportName, Province = Model.Province }).LocalDataSource("TestDataset", Model.Data).ControlSettings(settings).Attributes(new { Height = 750, Width = "100%", style = "border: none" })

I want use default export button of viewer and i want to change export filename with timestamp. Now, export filename is "TestReport", that isn't meaning. Please help me!

P/s: i don't want to create new action Excel Download as:

private ActionResult DownloadReport(ReportFormat format, bool isLocalReport, string filename = null)
        {
            if (isLocalReport)
            {
                return this.Report(
                    format,
                    LocalReportName,
                    new { Parameter1 = "Test", Parameter2 = 123 },
                    ProcessingMode.Local,
                    new Dictionary<string, DataTable>
                    {
                        { "Products", LocalData.GetProducts() },
                        { "Cities", LocalData.GetCities() }
                    },
                    filename);
            }

            return this.Report(
                format,
                RemoteReportName,
                new { Parameter1 = "Hello World!", Parameter2 = DateTime.Now, Parameter3 = 12345, Parameter4 = (DateTime?)null },
                filename: filename);
        }

**Comment by vivekcpatel** [comment]
I also would like to have a different report name when user clicks on export button. How could i do that?

**Comment by ilich** [comment]
I do not think you can change the report name whn uer clicks export button. This is the part of the original report viewer web control.

Could you send me a pull request for the proposed features, please?

**Comment by sven5** [comment]
It is possible to change the report file name through JavaScript. I'm currently using this because of the missing DisplayName property in 0.8.3. This is kind of a hack. I'm looking forward to have the DisplayName property in 0.8.4

Thanks
Sven
